src/heroku.ts

```diff
@@ -0,0 +1,70 @@
+import Heroku from 'heroku-client';
+import { retry } from '@lifeomic/attempt';
+
+interface HerokuIntegrationConfig {
+  apiKey: string;
+}
+
+let heroku: Heroku = undefined;
+
+export function getHerokuClient(config?: HerokuIntegrationConfig): Heroku {
```

Comment:
This function needs to be split since it acts as two different functions: an initialization function that takes a config and a getter function that does not. There is no time that I will call this without having to know which of those separate functions I want it to act as.

Reply:
Changed in #2